lib/watir/container.rb

```diff
@@ -36,6 +36,7 @@ def elements(*args)
     def extract_selector(selectors)
       case selectors.size
       when 2
+        Watir.logger.deprecate "Using ordered parameters to locate elements", "a Hash"
```

Comment:
To make it easier to find places that need corrections, can we include the selector in the message - eg `"Using ordered parameters, #{selectors}, to locate elements"` ?

Reply:
Looks great.